website/content/gateway/api_reference/changes/changes.md

```diff
@@ -10,6 +10,12 @@ Follow coming changes on the [source code repository](https://github.com/clearha
 
 Sorted by descending timestamp.
 
+#### Removal of `token[m4m][eci]` parameter
+Starting 2024-05-15, the `token[m4m][eci]` parameter has been removed. As the
+API accepts unknown parameters in general, this means that the parameter is now
+being ignored. It is recommended that the parameter is no longer sent to the
+transaction gateway.
+
```

Comment:
```suggestion
#### Removal of `token[m4m][eci]` parameter
Starting 2024-05-15, the `token[m4m][eci]` parameter will be ignored. As the
API accepts unknown parameters in general, this means that the parameter can still be sent. However it is recommended that the parameter be omitted to avoid confusion.

```

Reply:
https://github.com/clearhaus/gateway-api-docs/pull/192/commits/3dea5ece32c8209c43ab03b43ed9472699802308